spec/core/conditionals_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "logstash/devutils/rspec/spec_helper"
```

Comment:
I added a generic spec_helper, so we are sure even if we run a single test we can get a coverage report, otherwise not all rest requested the devutils spec_helper.

you can see this very same line in the general spec_helper.